clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -690,6 +690,20 @@ public Node provide() {
             this(false, callName, deadlineMs, nodeProvider);
         }
 
+        Call(String callName, long deadlineMs, NodeProvider nodeProvider,
+            int numTries, long nextAllowedTryMs) {
+            this(false, callName, deadlineMs, nodeProvider);
+            if (numTries < 0) {
+                throw new IllegalArgumentException("Number of tries for a call cannot be negative!");
```

Comment:
nit: I feel like a jerk for saying this, but can we avoid the exclamation points? The enthusiasm can be a bit vexing for an operator who hits this error.

Reply:
Fair point, will remove the exclamation mark.